be/src/testutil/parallel_test.h

```diff
@@ -22,4 +22,17 @@
 // in parallel.
 #define TOKENPASTE(x, y) x##y
 #define TOKENPASTE2(x, y) TOKENPASTE(x, y)
+
 #define PARALLEL_TEST(test_case_name, test_name) TEST(TOKENPASTE2(test_case_name, __LINE__), test_name)
+#define GROUP_PARALLEL_TEST(group_name, test_case_name, test_name) \
+    PARALLEL_TEST(test_case_name, group_name##_##test_name)
+
+#define GROUP_TEST_F(group_name, test_case_name, test_name) TEST_F(test_case_name, group_name##_##test_name)
+
+#ifdef NDEBUG
+#define GROUP_SLOW_PARALLEL_TEST(test_case_name, test_name) GROUP_PARALLEL_TEST(SLOW, test_case_name, test_name)
+#define GROUP_SLOW_TEST_F(test_case_name, test_name) GROUP_TEST_F(SLOW, test_case_name, test_name)
+#else
+#define GROUP_SLOW_PARALLEL_TEST(test_case_name, test_name) GROUP_PARALLEL_TEST(DISABLED, test_case_name, test_name)
```

Comment:
this will disable the cases by default under DEBUG/ASAN mode, right?

Reply:
The group of slow will be disabled, other group will not